tests/ftests/systemd.py

```diff
@@ -27,3 +29,91 @@ def is_delegated(config, scope_name):
                 # until we figure out something better :(
                 return True
             raise re
+
+    # This function builds the absolute path of the cgroup, using the default
+    # systemd path and checks if the cgroup exists. The function can build
+    # path for all three cgroup setup modes, including empty cgroup v2.
+    @staticmethod
+    def cgroup_exists(config, ctrl_name, cgroup_name, ignore_systemd=False):
+        # 'None' is a valid controller name, while creating an empty cgroup
+        # on cgroup v2 is unified setup. But it doesn't have a mount point,
+        # lets use the first available controller mount point to form the path
+        if ctrl_name is None:
+            if ignore_systemd:
+                raise IndexError('Unsupported controller \'None\'')
+
+            controllers = ['cpu', 'memory', 'io', 'pids', 'cpuset']
+            for controller in controllers:
+                ctrl_mnt = Cgroup.get_controller_mount_point(controller)
+                if 'Unknown version' not in ctrl_mnt:
```

Comment:
I don't think this `if` is buying you anything.  Unknown version and unknown mount point are raised as Exceptions, and thus I don't think this `if` is protecting you.